CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+Contributing to WALA
+====================
+
+WALA welcomes contributions of all kinds and sizes. This includes everything from from simple bug reports to large features.
+
+Workflow
+--------
+
+We love GitHub issues!
+
+For small feature requests, an issue first proposing it for discussion or demo implementation in a PR suffice.
+
+For big features, please open an issue so that we can agree on the direction, and hopefully avoid investing a lot of time on a feature that might need reworking.
+
+Small pull requests for things like typos, bug fixes, etc are always welcome.
+
+CI checks
```

Comment:
“checks” → “Checks” for capitalization consistent with other headings in this document.